src/Components/LogTable.css

```diff
@@ -15,6 +15,27 @@
 		position: relative;
 		word-wrap: break-word;
 		border-bottom: 1px solid var(--color-border);
+
+		button {
+			display: flex;
+			border: none !important;
+			background: none;
```

Comment:
Just noticed, that selector now affects all buttons in `<td>` also, so 'icon-clippy' is no longer visible on hover. Maybe move it to line 8?